zerver/lib/export.py

```diff
@@ -471,6 +472,13 @@ def get_realm_config() -> Config:
         parent_key='user_profile__in',
     )
 
+    Config(
+        table='zerver_reaction',
+        model=Reaction,
+        normal_parent=user_profile_config,
+        parent_key='user_profile__in',
+    )
```

Comment:
I'm not sure this is the right filtering structure; we want only the Reaction objects that go with messages that we're exporting, and this seems to grab all those from users.  E.g. if we're excluding private messages from the export, we don't want the reactions for those private messages to be included!

Reply:
`export_attachment_table` is a good thing to crib off of for this.